scripts/cybershake/install_cybershake.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+from scripts.cybershake.install_cybershake_fault import install_fault
+
+
+def main():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "path_cybershake",
+        type=str,
+        default=None,
+        help="the path to the root of a specific version cybershake.",
+    )
+    parser.add_argument(
+        "version", type=str, default="16.1", help="Please specify GMSim version"
+    )
+    parser.add_argument(
+        "fault_selection_list", type=str, default=None, help="The fault selection file"
+    )
+    parser.add_argument(
+        "--seed", type=str, default=0, help="The seed to be used for HF simulations. Default is to request a random seed."
+    )
+
+    args = parser.parse_args()
+
+    faults = {}
+    with open(args.fault_selection_file) as fault_file:
```

Comment:
This argument doesn't exists, I'm guessing this should be "fault_selection_file"?